mesa/space.py

```diff
@@ -989,22 +986,20 @@ def is_cell_empty(self, node_id: int) -> bool:
         return not self.G.nodes[node_id]["agent"]
 
     def get_cell_list_contents(self, cell_list: list[int]) -> list[GridContent]:
-        """Returns the contents of a list of cells ((x,y) tuples)
-        Note: this method returns a list of `Agent`'s; `None` contents are excluded.
+        """Returns a list of the agents in the nodes identified in `cell_list`;
```

Comment:
It's definitely better. Just to be sure, I'd say "returns a list of the agents contained in the nodes identified in `cell_list`".

Reply:
perfect, done